css/css-backgrounds/css3-border-image-repeat-repeat.html

```diff
@@ -1,27 +1,26 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>CSS Border Test:border image repeat property with value repeat</title>
-    <link rel="author" title="yanshasha" href="mailto:yanshasha133@gmail.com" />
-    <link rel="reviewer" title="Dayang Shen" href="mailto:shendayang@baidu.com" /> <!-- 2013-08-26 -->
-    <link rel="help" href="http://www.w3.org/TR/css3-background/#border-image-repeat" />
-    <link rel="match" href="reference/css3-border-image-repeat-repeat-ref.html" />
-    <style type="text/css">
+
+    <meta charset="UTF-8">
+    <title>CSS Border Test: border image repeat property with value repeat</title>
+    <link rel="author" title="yanshasha" href="mailto:yanshasha133@gmail.com">
+    <link rel="reviewer" title="Dayang Shen" href="mailto:shendayang@baidu.com"> <!-- 2013-08-26 -->
+    <!--
+    corrected and improved to be more precise
+    by Gérard Talbot 2020-05-27
+    -->
+    <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-repeat">
+    <link rel="match" href="reference/css3-border-image-repeat-repeat-ref.html">
+    <style>
             div {
-                width: 100px;
-                height: 100px;
-                border: solid 26px blue;
-                border-image-source: url(support/border.png);
-                border-image-slice: 26;
+                border: red solid 27px;
                 border-image-repeat: repeat;
+                border-image-slice: 27;
+                border-image-source: url("support/blue-and-red-diamonds-81x81.png");
+                height: 81px;
+                width: 81px;
```

Comment:
If I'm understanding this test correctly, this will result in the border-image a) not being resized and b) repeating a whole number of times. That's not a rigorous enough test. We need to ensure that there is some resizing happening (preferably resizing down by a factor of three, so the resizing lands on pixel boundaries) to test that resizing happens correctly, and we need to use a non-whole number of tiles to ensure that the behavior we are seeing is proper tiling and not rounding or something else.